src/sentry/snuba/metrics/query.py

```diff
@@ -139,7 +140,7 @@ def validate_orderby(self) -> None:
             metric_mri = get_mri(f.field.metric_name)
             # Construct a metrics expression
             metric_field_obj = metric_object_factory(f.field.op, metric_mri)
-            entity = metric_field_obj.get_entity(self.project_ids)
+            entity = metric_field_obj.get_entity(self.project_ids, UseCaseKey.RELEASE_HEALTH)
```

Comment:
This is not necessarily true because this could be also a query for performance, so you need to propagate here the usecase from the request calling this somehow

Reply:
I know I said we shouldn't add the use case here https://github.com/getsentry/sentry/pull/36308/files#diff-7da0855ce9cf928e32249842b19bdcec279a052ee9fcc9b9583f2ff3fb7fea33L80 but if there is no other way to propagate it then maybe just add it as a parameter here. I would spend sometime thinking how to propagate this down without adding it to class it though